fixtures/dom/src/index.test.js

```diff
@@ -25,153 +24,210 @@ function App(props) {
   return 'hello world';
```

Comment:
'hello world' is a bit tired and overused. What about 'hello nerds'? I think the term 'nerd' now has a much more positive connotation, and I personally think it's endearing. Just trying to come up with a random string of text to test out this comment feature!  

```suggestion
  return 'hello nerds';
```

Reply:
Thanks for the feedback! I've committed your suggestion.